selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/DeleteOrganizationTest.java

```diff
@@ -29,97 +32,91 @@
 import org.testng.annotations.Test;
 
 /**
- * Test validates organization deletion.
+ * Test validates organization deletion from the Settings tab by the Delete button
  *
  * @author Ann Shumilova
  */
 public class DeleteOrganizationTest {
+  private static final String PARENT_ORG_NAME = generate("organization", 5);
+  private static final String CHILD_ORG_NAME = generate("organization", 5);
+
+  private List<OrganizationDto> organizations = new ArrayList<>();
   private OrganizationDto parentOrganization;
   private OrganizationDto childOrganization;
 
+  @Inject
+  @Named("admin")
+  private TestOrganizationServiceClient testOrganizationServiceClient;
+
   @Inject private OrganizationListPage organizationListPage;
   @Inject private OrganizationPage organizationPage;
   @Inject private NavigationBar navigationBar;
   @Inject private ConfirmDialog confirmDialog;
   @Inject private Dashboard dashboard;
-
-  @Inject
-  @Named("admin")
-  private TestOrganizationServiceClient testOrganizationServiceClient;
-
   @Inject private TestUser testUser;
-  @Inject private AdminTestUser adminTestUser;
 
   @BeforeClass
   public void setUp() throws Exception {
-    parentOrganization =
-        testOrganizationServiceClient.create(NameGenerator.generate("organization", 5));
-    childOrganization =
-        testOrganizationServiceClient.create(NameGenerator.generate("organization", 5));
-
+    parentOrganization = testOrganizationServiceClient.create(PARENT_ORG_NAME);
+    childOrganization = testOrganizationServiceClient.create(CHILD_ORG_NAME);
     testOrganizationServiceClient.addAdmin(parentOrganization.getId(), testUser.getId());
     testOrganizationServiceClient.addAdmin(childOrganization.getId(), testUser.getId());
+    organizations.add(parentOrganization);
+    organizations.add(childOrganization);
 
     dashboard.open(testUser.getName(), testUser.getPassword());
   }
 
   @AfterClass
   public void tearDown() throws Exception {
-    testOrganizationServiceClient.deleteById(childOrganization.getId());
-    testOrganizationServiceClient.deleteById(parentOrganization.getId());
+    for (OrganizationDto organization : organizations)
+      testOrganizationServiceClient.deleteById(organization.getId());
```

Comment:
Such code can affect organizations which are created in parallel threads of test execution.

Reply:
The **Organizations** list includes only organizations from this test.

Reply:
I see.
`organizations` field is used for tearDown() method only.
To simplify the code, we could avoid usage of `organizations` field at all and to remove two organizations explicitly instead:
```javascript
testOrganizationServiceClient.deleteById(childOrganization.getId());
testOrganizationServiceClient.deleteById(parentOrganization.getId());
```